.gitignore

```diff
@@ -23,4 +23,5 @@
 *.swp
 temp
 results
-bench/benchmarks*
\ No newline at end of file
+bench/benchmarks*
+vendor
```

Comment:
what's the intention of this folder?

Reply:
I prefer `bundle install --path vendor/bundle` to global installs. One more path in `.gitignore` doesn't hurt, does it?

Reply:
nope was just curious